### DISCUSSION TITLE
How to inject an element to index.template.html upon build?

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#10420] [General - Components / Directives / etc]
I would like to add an element to `index.template.html`, only when doing `quasar build` (not `quasar dev`).

Is this made possible by Quasar?

**Comment by ontwikkelfabriek** [comment]
@wsw70 

>I would like to add an element to index.template.html, only when doing quasar build (not quasar dev).


This is a Qv1 example but I think you can easily convert it to Qv2:

```javascript
// index.template.html
<% if (htmlWebpackPlugin.options.ctx.prod) { %>
        <script src="statics/js/meteor.bundle.dev.js"></script>
<% } %>
```
